src/snowflake/connector/auth/_auth.py

```diff
@@ -127,8 +127,9 @@ def base_auth_data(
         internal_application_name,
         internal_application_version,
         ocsp_mode,
-        login_timeout,
-        network_timeout=None,
+        login_timeout=None,
+        request_timeout=None,
+        socket_timeout=None,
```

Comment:
Type-hint at least your new arguments, but if you could do all of them that'd be great

Reply:
Added type hints for my own, but I'm not entirely sure about all the existing ones.